applications/FemToDemApplication/fem_to_dem_application.cpp

```diff
@@ -59,6 +59,8 @@ void KratosFemToDemApplication::Register()
  	KratosApplication::Register();
 	
 	//REGISTER VARIABLES FEM2DEM
+	KRATOS_REGISTER_VARIABLE(DEM_PARTICLE_POINTER)

```

Comment:
Why did you commented the register but it is still here?

Reply:
I use this var but it didn't compile if I register it in Python IDK why...